raiwidgets/raiwidgets/responsibleai_dashboard_input.py

```diff
@@ -101,9 +106,11 @@ def matrix(self, data):
         except Exception as e:
             print(e)
             traceback.print_exc()
+            e_str = _format_exception(e)
             return {
                 WidgetRequestResponseConstants.error:
-                    "Failed to generate json matrix representation",
+                    "Failed to generate json matrix representation,"
+                    "inner error: {}".format(e_str),
```

Comment:
I'm a bit worried if we return these in some other platform that they may reveal private user information, like column names etc, so anything that calls these will need to sanitize them?

Reply:
it seems this is fine after discussion because it's not logged anywhere anyway

Reply:
Talked offline, in AzureML, we will make sure we don't log any detailed error message which could contain user's private information.